src/randoop/JunitFileWriter.java

```diff
@@ -153,6 +143,19 @@ private File writeSubSuite(List<ExecutableSequence> sequencesForOneFile, String
     return file;
   }
 
+/*
+ * Writes a test method to the output stream for the sequence s with name "test"+testCounter. 
+ */
+  private void writeTest(PrintStream out, String className, int testCounter, ExecutableSequence s) {
+    out.println("  @Test");
+    out.println("  public void test" + testCounter + "() throws Throwable {");
+    out.println();
+    out.println(indent("if (debug) { System.out.println(); System.out.print(\"" + className + ".test" + testCounter + "\"); }"));
```

Comment:
It's weird that two spaces are used for two-space indent but indent() is used for 4-space indent.  I can see the need for indent() when its argument was generated elsewhere and has multiple lines, but it doesn't make so much sense to me here.  Maybe it's for consistency with the occurrence two lines below, which is necessary?